utils/rpms/daos.spec

```diff
@@ -244,8 +244,6 @@ Requires: python3-tabulate
 Requires: python3-defusedxml
 %endif
 Requires: fio
-Requires: meson
```

Comment:
Doesn't any change to the spec file warrant a version bump + %changelog entry?

Reply:
Yes, I'm waiting for my other ftest PR to land, then I can rebase this one and make the required RPM changes.  At the minute I'm looking for any more feedback on the rest of it so I could address that now.

Reply:
Done